source/_references/stacks.md

```diff
@@ -447,25 +447,31 @@ if (true === $deleted) {
 It is possible to list all stacks for an organization.
 
 ```language-bash
-curl -X GET 'https://api.rokka.io/stacks/testorganization'
+curl -X GET 'https://api.rokka.io/stacks/testorganization' -H 'Api-Key: key'
 ```
 
-```language-php
-use Rokka\Client\Core\Stack;
-
-$client = \Rokka\Client\Factory::getImageClient('testorganization', 'apiKey', 'apiSecret');
-
```

Comment:
why did you remove the PHP example? We don't have any other place, where the PHP API is described in full (may be something we should do some day, of course)